src/utils/crypto.ts

```diff
@@ -0,0 +1,26 @@
+import CryptoJS from 'crypto-js';
+import { Credential } from '../types';
+
+export function deCrypt(credential: Credential, key: string): Credential {
+  const deCryptPassword = CryptoJS.TripleDES.decrypt(
+    credential.password,
+    key
+  ).toString(CryptoJS.enc.Utf8);
+  const partiallyDeCryptedCredential = {
+    ...credential,
+    password: deCryptPassword,
+  };
+  return partiallyDeCryptedCredential;
+}
+
+export function enCript(credential: Credential, key: string): Credential {
```

Comment:
Typo and same as the above
```suggestion
export function encrypt(credential: Credential, key: string): Credential {
```